packages/closer/components/ProjectCard/ProjectCard.tsx

```
@@ -0,0 +1,87 @@
+          </Heading>
+          <p>{project.description}</p>
+        </div>
+        <div className="flex flex-col gap-6">
+          <div className="flex flex-col gap-2 text-md">
+            <div className="flex gap-2 flex-wrap pb-2">
+              {project?.skills &&
+                project?.skills.map((skill) => (
+                  <Tag key={skill} size="small" color="primary">
+                    {skill}
+                  </Tag>
+                ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)